build/JSAssetPlayCanvas.js

```diff
@@ -0,0 +1,68 @@
+var Preprocessor = require("preprocessor");
+const JSAsset = require("parcel-bundler/src/assets/JSAsset");
+//const { parse } = require("ifdef-loader/preprocessor");
+
+/*
+JSAssetPlayCanvas JSAssetPlayCanvas {
+  id: 'framework/components/registry.js',
+  name: 'C:\\web\\playcanvas-engine\\src\\framework\\components\\registry.js',
```

Comment:
I'm sure you are well aware of all this, but this script is very specific to your machine. 😄